config/application.yml.example

```diff
@@ -58,11 +58,6 @@ development:
   system_satisfaction_survey: false
   editable_statuses: { 14: ['first_draft', 'draft', 'submitted', nil, 'get_a_cost_estimate', 'awaiting_pi_approval'],
                      48: ['draft', 'submitted'] }
```

Comment:
I wasn't aware of the new Dashboard change when this pull request was submitted. With that in mind, let me rework on this.